sdk/metric/aggregation/aggregation.go

```diff
@@ -164,3 +164,65 @@ func (h ExplicitBucketHistogram) Copy() Aggregation {
 		NoMinMax:   h.NoMinMax,
 	}
 }
+
+// ExponentialHistogram is an aggregation that summarizes a set of
+// measurements as an histogram with buckets widths that grow exponentially.
+type ExponentialHistogram struct {
+	// MaxSize is the maximum number of buckets to use for the histogram.
+	MaxSize int
+	// MaxScale is the maximum resolution scale to use for the histogram.
+	MaxScale int
+	// ZeroThreshold sets the minimum value blow which will be recorded as zero.
+	ZeroThreshold float64
+
+	// NoMinMax indicates whether to not record the min and max of the
+	// distribution. By default, these extrema are recorded.
+	//
+	// Recording these extrema for cumulative data is expected to have little
+	// value, they will represent the entire life of the instrument instead of
+	// just the current collection cycle. It is recommended to set this to true
+	// for that type of data to avoid computing the low-value extrema.
+	NoMinMax bool
+}
+
+// DefaultExponentialHistogram returns the default Exponential Histogram aggregation.
+func DefaultExponentialHistogram() ExponentialHistogram {
+	return ExponentialHistogram{
+		MaxSize:       160,
+		MaxScale:      20,
+		ZeroThreshold: 0.0,
+	}
+}
```

Comment:
I did this because MaxScale of 0 is valid, but probably not what users want if they were to use an `ExponentialHistogram{}`.  If there is a sane alternative, provided by a `Default` then we can cut down users stumbling.

Reply:
I will remove this, but continue the discussion in https://github.com/open-telemetry/opentelemetry-go/issues/4347.